docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,85 @@
+    mediatype: image/png
+  version: 0.0.1
+  provider:
+    name: Red Hat, Inc.
+  maturity: alpha
```

Comment:
Please omit

Reply:
@aravindhp why should we omit this? Last I heard this is one of the fields of the details page of the operator in the UI. Has this changed?

Reply:
We omit this until we have a process in place to define it.